packages/admin/src/brevoContacts/form/BrevoContactForm.tsx

```diff
@@ -42,9 +42,13 @@ import {
 } from "./BrevoContactForm.gql.generated";
 
 export type EditBrevoContactFormValues = {
+    [key: string]: unknown;
+};
+
+type BaseBrevoContactFormValues = EditBrevoContactFormValues & {
+    //Umbenennen
```

Comment:
Should hits still be renamed? Or can the comment be deleted?

Reply:
Now that I think about it: I do not think `Base` is not really the correct name for this. Since this extends the BaseFormValues (defined in `& { ...}`} with the configured values. Something like `BrevoContactFormValuesWithAttributes` would be easier to understand I think. What do you think?

Reply:
Maybe `EditBrevoContactFormValuesWithAttributes`?

Reply:
👍🏻